pkg/cli/config.go

```diff
@@ -95,20 +95,44 @@ func BaseConfig() *Config {
 	}
 }
 
-func LoadConfig(srcs []string, c *Config) error {
-	combined := []byte{}
-	for _, path := range srcs {
-		b, err := os.ReadFile(path)
+func LoadConfig(sysSrc, userSrc string, c *Config) error {
+	loadFn := func(src string, out *Config) error {
+		if src == "" {
+			return nil
+		}
+		b, err := os.ReadFile(src)
 		if err != nil {
-			return fmt.Errorf("error reading config file: %w", err)
+			return err
+		}
+		decoder := toml.NewDecoder(bytes.NewReader(b))
+		// Fail if there is some unknown configuration. This is better than silently
+		// ignoring a (perhaps misspelled) config entry.
+		decoder.Strict(true)
+		if err := decoder.Decode(out); err != nil {
+			return err
 		}
-		combined = append(combined, b...)
+		return nil
+	}
+	// Load system configuration
+	if err := loadFn(sysSrc, c); err != nil {
+		return fmt.Errorf("error loading system configuration: %w", err)
+	}
+	if userSrc == "" {
+		return nil
 	}
-	decoder := toml.NewDecoder(bytes.NewReader(combined))
-	// Fail if there is some unknown configuration. This is better than silently
-	// ignoring a (perhaps misspelled) config entry.
-	decoder.Strict(true)
-	return decoder.Decode(c)
+	tmpSysConfig := &Config{}
+	if err := loadFn(sysSrc, tmpSysConfig); err != nil {
+		return fmt.Errorf("error loading system configuration: %w", err)
+	}
```

Comment:
Instead of loading system configs twice, would it be possible to store the `Services` dictionary from the first time it was loaded?

Reply:
Done.